### PR TITLE
Rework user guide #78082

### LINKED ISSUE #78082: Ansible User Guide can be separated out for better retrievability
### Summary

The Ansible User Guide crams in too content and high-level organization is confusing. As a new user I find it difficult to parse through and identify suitable content for my goals.

It would probably help to break the User Guide into more logically organized sections and reduce the amount of overall clutter.

### Issue Type

Documentation Report

### Component Name

docs/docsite/rst/user_guide

### Ansible Version

```console
$ ansible --version
ansible [core 2.12.6]
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/home/dnaro/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3.10/site-packages/ansible
  ansible collection location = /home/dnaro/.ansible/collections:/usr/share/ansible/collections
  executable location = /usr/bin/ansible
  python version = 3.10.4 (main, Mar 25 2022, 00:00:00) [GCC 12.0.1 20220308 (Red Hat 12.0.1-0)]
  jinja version = 3.0.3
  libyaml = True
```

### Configuration

```console
# if using a version older than ansible-core 2.12 you should omit the '-t all'
$ ansible-config dump --only-changed -t all
```

### OS / Environment

Fedora 36

### Additional Information

When this improvement is merged it will be easier to find content in the User Guide.

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

### PR DESCRIPTION
**Pull request by oraNod**
##### SUMMARY
Fixes #78082 

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
User Guide

##### ADDITIONAL INFORMATION
Reworking the User Guide.

### CODE REVIEW

File: docs/docsite/rst/core_index.rst [hunk 1]
```diff
@@ -49,6 +49,9 @@ This documentation covers the version of ``ansible-core`` noted in the upper lef
    :caption: Using Ansible Core
 
    user_guide/index
+   playbook_guide/index
+   tips_tricks/index
+   win_guide/index
```
Comment (samccann) [inline docs/docsite/rst/core_index.rst:54]: You'll need to make these same changes to ansible_index.rst as well. core_index reflects the core docsite (docs.ansible.com/ansible-core) - aka docs w/o all the collection level docs.  ansible_index is the Ansible (the package) docs that go to docs.ansible.com/ansible.

**Comment by samccann** [comment]
I put it up on test if anyone wants to take a look - http://docs.testing.ansible.com/ansible-core/2.14/

In general, we try not to do PRs this big. So for example, an easier approach to review/approve/write redirects for, would have been to move one guide at a time per PR.  So like split out the windows stuff in one pr, etc.  Backporting a PR this big could be problematic.

**Comment by samccann** [comment]
I put some long diatribe on the issue we should consider as well ;-)

File: docs/docsite/rst/core_index.rst (hunk 1 above)
Comment (oraNod) [reply docs/docsite/rst/core_index.rst:54]: Thanks, I'll do that.

**Pull request closed without merging**